OracleGoldenGate/createDB.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
I think what you should do is call `createDb.sh` and afterwards have your own script, e.g. "setupGoldenGate.sh" that will perform those steps that you have outlined above. The run script will detect whether it has to create the DB or it's already there so from there I think you are fine. But I did spot one thing with the `runOracle.sh` script. The hard coded `tail -f` at the end. I will have to enhance the `runOracle.sh` script to allow for batch mode, i.e. not wait and tail on the alert log but just finish. That would allow you to just execute `createDb.sh`, `setupGoldenGate.sh` and `runOracle.sh` and then proceed with whatever is appropriate for your container.